boilerplate/openshift/golang-osd-operator/csv-generate/csv-generate.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash -x
```

Comment:
This would replace app_sre_build_image_catalog.sh?

Reply:
The 3 shell scripts form `csv-generate` folder are replacing altogether the `app_sre_build_image_catalog.sh`